root_locals.tf

```diff
@@ -24,7 +24,6 @@ locals {
   athena_bucket     = "${var.project}-athena-${local.environment}"
   config_bucket     = "${var.project}-config-${local.environment}"
 
-  hosted_zone_envs    = ["intg", "staging", "prod"]
-  create_domain_email = contains(local.hosted_zone_envs, terraform.workspace) ? true : false
-  create_hosted_zone  = contains(local.hosted_zone_envs, terraform.workspace) ? true : false
+  create_domain_email = var.project == "tdr" ? true : false
```

Comment:
Is this going to work with the TDR sandbox and management accounts. With the current configuration the domain email and hosted zone are not created in these two environments. With the suggested changes they will be created, which is not desired behaviour.

Reply:
@TomJKing both the TDR sandbox and management accounts have hosted zones created by this repo - with the previous conditional they would have been destroyed.

Reply:
I've logged in to each of the 4 Digital Preservation AWS accounts, and none of them contain a Route53 Hosted Zone

Reply:
I've also confirmed the behaviour by doing a Terraform plan of this branch against TDR sandbox and there are no changes planned to hosted zones